### DISCUSSION TITLE
(ModuleNotFoundError) cv2

### DISCUSSION THREAD

**Discussion by ghost** [apache/airflow#41970] [Q&A]
I have used docker desktop to install airflow. And I have copied some sample dags available on localhost to my vscode environment and that code works fine. But when I tried to apply my python code that has module cv2 imported , the dag was broken. Module is not found appeared in the log. Other modules are successfully imported. What can I do?

![image](https://github.com/user-attachments/assets/8cdd3dae-0cd9-4750-8471-216b74499323)

**Comment by potiuk** [comment]
Read the docs: https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html#using-custom-images
